edx_event_bus_kafka/consumer/tests/test_event_consumer.py

```diff
@@ -65,44 +71,54 @@ def setUp(self):
                 )
             )
         }
+        self.signal_type_bytes = b'org.openedx.learning.auth.session.login.completed.v1'
+        self.signal_type = self.signal_type_bytes.decode('utf-8')
         self.normal_message = FakeMessage(
             topic='user_stuff',
             headers=[
-                ['ce_type', b'org.openedx.learning.auth.session.login.completed.v1']
+                ['ce_type', self.signal_type_bytes]
             ],
             key=b'\x00\x00\x00\x00\x01\x0cfoobob',  # Avro, as observed in manual test
             value=self.normal_event_data,
             error=None,
         )
+        self.mock_signal = Mock(event_type=self.signal_type, init_data={})
+        self.event_consumer = KafkaEventConsumer('test_topic', 'test_group_id', self.mock_signal)
 
     def test_emit(self):
-        mock_signal = Mock()
-        with patch.object(OpenEdxPublicSignal, 'get_signal_by_type', return_value=mock_signal) as mock_lookup:
-            ec.emit_signals_from_message(self.normal_message)
+        with patch.object(OpenEdxPublicSignal, 'get_signal_by_type', return_value=self.mock_signal) as mock_lookup:
+            self.event_consumer.emit_signals_from_message(self.normal_message)
 
-        mock_lookup.assert_called_once_with('org.openedx.learning.auth.session.login.completed.v1')
-        mock_signal.send_event.assert_called_once_with(**self.normal_event_data)
+        mock_lookup.assert_called_once_with(self.signal_type)
+        self.mock_signal.send_event.assert_called_once_with(**self.normal_event_data)
 
     def test_no_type(self):
         msg = copy.copy(self.normal_message)
         msg._headers = []
 
         with patch.object(OpenEdxPublicSignal, 'get_signal_by_type') as mock_lookup:
-            ec.emit_signals_from_message(msg)
+            self.event_consumer.emit_signals_from_message(msg)
 
         mock_lookup.assert_not_called()
 
     def test_unknown_type(self):
+        # If we pretend that the test signal type is not a real one, behave accordingly.
+        with patch.object(OpenEdxPublicSignal, 'get_signal_by_type', side_effect=KeyError('not found')) as mock_lookup:
+            # Should just suppress exception and log
+            self.event_consumer.emit_signals_from_message(self.normal_message)
+
+        mock_lookup.assert_called_once_with(self.signal_type)
+        assert not self.mock_signal.send_event.called
+
+    def test_mismatching_types(self):
```

Comment:
Maybe clearer? Could also change `xxxx` to `some.other.signal`.
```suggestion
    def test_unwanted_types(self):
```